org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
```

Comment:
why do you need the GNOME runtime?

Reply:
because it contains gstreamer

Reply:
the docs says:
https://docs.flatpak.org/en/latest/available-runtimes.html
The GNOME runtime is appropriate for any application that uses the GNOME platform. It is based on the Freedesktop runtime and adds the GNOME platform, including:
GStreamer.

is the docs wrong?

Reply:
gstreamer is in freedesktop SDK. so might might be wrong. It has been for a few years now. For once you follow the doc to the letter...

Reply:
these are all the GStreamer packages in Freedesktop SDK... (unless they don't have `gst` in their name)

```
gstcdg                         gstcdg - GStreamer CDG codec Plugin
gstclaxon                      gstclaxon - GStreamer Claxon FLAC Decoder Plugin
gstdav1d                       gstdav1d - GStreamer dav1d AV1 decoder Plugin
gstfallbackswitch              gstfallbackswitch - GStreamer Fallback Switcher and Source Plugin
gstfmp4                        gstfmp4 - GStreamer Fragmented MP4 Plugin
gstgif                         gstgif - GStreamer GIF plugin
gstgopbuffer                   gstgopbuffer - Store complete groups of pictures at a time
gsthlssink3                    gsthlssink3 - GStreamer HLS (HTTP Live Streaming) Plugin
gsthsv                         gsthsv - GStreamer plugin with HSV manipulation elements
gstjson                        gstjson - GStreamer JSON Plugin
gstlewton                      gstlewton - GStreamer lewton Vorbis Decoder Plugin
gstlivesync                    gstlivesync - Livesync Plugin
gstmp4                         gstmp4 - GStreamer Rust MP4 Plugin
gstmpegtslive                  gstmpegtslive - GStreamer MPEG-TS Live sources
gstndi                         gstndi - GStreamer NewTek NDI Plugin
gstoriginalbuffer              gstoriginalbuffer - GStreamer Origin buffer meta Plugin
gstraptorq                     gstraptorq - GStreamer RaptorQ FEC Plugin
gstrav1e                       gstrav1e - GStreamer rav1e AV1 Encoder Plugin
gstreamer-1.0                  gstreamer-1.0 - Streaming media framework
gstreamer-allocators-1.0       gstreamer-allocators-1.0 - Allocators implementation
gstreamer-analytics-1.0        gstreamer-analytics-1.0 - GStreamer analytics metadata library
gstreamer-app-1.0              gstreamer-app-1.0 - Helper functions and base classes for application integration
gstreamer-audio-1.0            gstreamer-audio-1.0 - Audio helper functions and base classes
gstreamer-bad-audio-1.0        gstreamer-bad-audio-1.0 - Bad audio library for GStreamer elements
gstreamer-base-1.0             gstreamer-base-1.0 - Base classes for GStreamer elements
gstreamer-check-1.0            gstreamer-check-1.0 - Unit testing helper library for GStreamer modules
gstreamer-codecparsers-1.0     gstreamer-codecparsers-1.0 - Bitstream parsers for GStreamer elements
gstreamer-controller-1.0       gstreamer-controller-1.0 - Dynamic parameter control for GStreamer elements
gstreamer-cuda-1.0             gstreamer-cuda-1.0 - Unstable library to work with CUDA inside GStreamer
gstreamer-fft-1.0              gstreamer-fft-1.0 - FFT implementation
gstreamer-gl-1.0               gstreamer-gl-1.0 - Streaming media framework, OpenGL plugins libraries
gstreamer-gl-egl-1.0           gstreamer-gl-egl-1.0 - Streaming media framework, OpenGL plugins libraries (EGL specifics)
gstreamer-gl-prototypes-1.0    gstreamer-gl-prototypes-1.0 - Streaming media framework, OpenGL plugins libraries (OpenGL Prototypes)
gstreamer-gl-wayland-1.0       gstreamer-gl-wayland-1.0 - Streaming media framework, OpenGL plugins libraries (Wayland specifics)
gstreamer-gl-x11-1.0           gstreamer-gl-x11-1.0 - Streaming media framework, OpenGL plugins libraries (X11 specifics)
gstreamer-insertbin-1.0        gstreamer-insertbin-1.0 - Bin to automatically and insertally link elements
gstreamer-mpegts-1.0           gstreamer-mpegts-1.0 - GStreamer MPEG-TS support
gstreamer-mse-1.0              gstreamer-mse-1.0 - GStreamer Support for W3C Media Source Extensions
gstreamer-net-1.0              gstreamer-net-1.0 - Network-enabled GStreamer plug-ins and clocking
gstreamer-pbutils-1.0          gstreamer-pbutils-1.0 - General utility functions
gstreamer-photography-1.0      gstreamer-photography-1.0 - High level API for transcoding using GStreamer
gstreamer-play-1.0             gstreamer-play-1.0 - GStreamer Player convenience library
gstreamer-player-1.0           gstreamer-player-1.0 - GStreamer Player convenience library
gstreamer-plugins-bad-1.0      gstreamer-plugins-bad-1.0 - Streaming media framework, bad plugins libraries
gstreamer-plugins-base-1.0     gstreamer-plugins-base-1.0 - Streaming media framework, base plugins libraries
gstreamer-riff-1.0             gstreamer-riff-1.0 - RIFF helper functions
gstreamer-rtp-1.0              gstreamer-rtp-1.0 - RTP base classes and helper functions
gstreamer-rtsp-1.0             gstreamer-rtsp-1.0 - RTSP base classes and helper functions
gstreamer-sctp-1.0             gstreamer-sctp-1.0 - SCTP helper functions
gstreamer-sdp-1.0              gstreamer-sdp-1.0 - SDP helper functions
gstreamer-tag-1.0              gstreamer-tag-1.0 - Tag base classes and helper functions
gstreamer-transcoder-1.0       gstreamer-transcoder-1.0 - High level API for transcoding using GStreamer
gstreamer-va-1.0               gstreamer-va-1.0 - GStreamer VA support
gstreamer-video-1.0            gstreamer-video-1.0 - Video base classes and helper functions
gstreamer-vulkan-1.0           gstreamer-vulkan-1.0 - GStreamer Vulkan support
gstreamer-vulkan-wayland-1.0   gstreamer-vulkan-wayland-1.0 - GStreamer Vulkan support (Wayland Specifics)
gstreamer-vulkan-xcb-1.0       gstreamer-vulkan-xcb-1.0 - GStreamer Vulkan support (XCB Specifics)
gstreamer-wayland-1.0          gstreamer-wayland-1.0 - GStreamer Wayland support
gstreamer-webrtc-1.0           gstreamer-webrtc-1.0 - GStreamer WebRTC support
gstreamer-webrtc-nice-1.0      gstreamer-webrtc-nice-1.0 - libnice based implementaion for GstWebRTCICE
gstregex                       gstregex - GStreamer Regular Expression Plugin
gstreqwest                     gstreqwest - GStreamer reqwest HTTP Source Plugin
gstrsaudiofx                   gstrsaudiofx - GStreamer Rust Audio Effects Plugin
gstrsclosedcaption             gstrsclosedcaption - GStreamer Rust Closed Caption Plugin
gstrsinter                     gstrsinter - GStreamer Inter Plugin
gstrsonvif                     gstrsonvif - GStreamer Rust ONVIF Plugin
gstrspng                       gstrspng - GStreamer Rust PNG encoder/decoder
gstrsrtp                       gstrsrtp - GStreamer Rust RTP Plugin
gstrsrtsp                      gstrsrtsp - GStreamer RTSP Client Plugin
gstrstracers                   gstrstracers - GStreamer Rust tracers plugin
gstrsvideofx                   gstrsvideofx - GStreamer Rust Video Effects Plugin
gstrswebp                      gstrswebp - GStreamer WebP Plugin
gstsodium                      gstsodium - GStreamer plugin for libsodium-based file encryption and decryption
gsttextahead                   gsttextahead - GStreamer Plugin for displaying upcoming text buffers ahead of time
gsttextwrap                    gsttextwrap - GStreamer Text Wrap Plugin
gstthreadshare                 gstthreadshare - GStreamer Threadshare Plugin
gsttogglerecord                gsttogglerecord - GStreamer Toggle Record Plugin
gsturiplaylistbin              gsturiplaylistbin - GStreamer Playlist Playback Plugin
```